mismi-core/test/Test/Mismi.hs

```diff
@@ -29,5 +32,5 @@ testAWS =
 runAWSDefaultRegion :: AWS a -> IO a
 runAWSDefaultRegion a = do
   r <- eitherT (const $ pure Sydney) pure getRegionFromEnv
-  e <- newEnv r Discover
+  e <- eitherT (fail . T.unpack . renderRegionError) pure discoverAWSEnv
```

Comment:
You're no longer using `r`, and this will fail if `AWS_DEFAULT_REGION` is not set.